rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -801,6 +803,42 @@ public void testCancel() {
         runLRA(true);
     }
 
+    /**
+     * This test checks that LRA can be used with a JDBC Object Store
+     */
+    @Test
+    public void jdbcStoreTest() {
+
+        BeanPopulator.getDefaultInstance(ObjectStoreEnvironmentBean.class).setObjectStoreType(JDBCStore.class.getName());
+        String objectStoreType = BeanPopulator.getDefaultInstance(ObjectStoreEnvironmentBean.class).getObjectStoreType();
+
+        LRALogger.logger.infof("The Object Store type is: %s", objectStoreType);
+
+        URI lraIdUri = lraClient.startLRA("ObjectStoreTestCase#newLRA");
+
+        assertNotNull("LRA should have been added to the object store", lraIdUri);
+
+        assertEquals("Expected Active", LRAStatus.Active, getStatus(lraIdUri));
+    }
+
+    /**
+     * This test checks that LRA can be used with a ShadowNoFileLockStore Object Store
+     */
+    @Test
+    public void shadowNoFileLockStoreTest() {
```

Comment:
Why did you choose to test the file system store but none of the others. I would have expected a test against just the JDBC store or all of the supported ones - on wildfly we support the JDBC, filesystem and journal stores. But, yes, I like the idea of sanity checking all of the supported stores.

Reply:
Yeah, it is true, I should have checked other object store types. I'll update the PR :-)